src/tasks/che.ts

```diff
@@ -675,7 +675,20 @@ export class CheTasks {
 
           const cheUrl = await this.che.cheURL(flags.chenamespace)
           messages.push(`Users Dashboard           : ${cheUrl}`)
-          messages.push('Admin user login          : "admin:admin". NOTE: must change after first login.')
+
+          const cr = await this.kube.getCheCluster(flags.chenamespace)
+          if (ctx.isOpenShift && cr && cr.spec && cr.spec.auth && cr.spec.auth.openShiftoAuth) {
```

Comment:
```suggestion
          if (ctx.isOpenShift4 && cr && cr.spec && cr.spec.auth && cr.spec.auth.openShiftoAuth) {
```

Reply:
Agreed

Reply:
No.
The main point is to print `admin:admin` only of OAuth isn't enable and HTPasswd user credentials will be printed only of secret is created.